sdk/template/azure_template/src/lib.rs

```diff
@@ -0,0 +1,14 @@
+pub fn add(left: u64, right: u64) -> u64 {
```

Comment:
I know this is generated initially, but we should still put the copyright headers in the source files so someone doesn't forget later. A couple of us have plans to put more in here for the very purpose of treating this crate as a canary. /cc @HarshaNalluru